x-pack/plugins/synthetics/e2e/journeys/synthetics/global_parameters.journey.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { journey, step, before, after, expect } from '@elastic/synthetics';
+import { cleanTestParams } from './services/add_monitor';
+import { syntheticsAppPageProvider } from '../../page_objects/synthetics_app';
+
+journey(`GlobalParameters`, async ({ page, params }) => {
+  const syntheticsApp = syntheticsAppPageProvider({ page, kibanaUrl: params.kibanaUrl });
+
+  before(async () => {
+    await cleanTestParams(params);
+  });
+
+  after(async () => {
+    await cleanTestParams(params);
+  });
+
+  step('Go to Settings page', async () => {
+    await syntheticsApp.navigateToSettings(true);
+  });
+
+  step('go to params tab', async () => {
+    await page.click('text=Global Parameters');
+  });
+
+  step('Click text=Settings', async () => {
+    await page.click('text=Settings');
+    expect(page.url()).toBe('http://localhost:5620/app/synthetics/settings/alerting');
+  });
+  step('Click text=Global Parameters', async () => {
+    await page.click('text=Global Parameters');
+    expect(page.url()).toBe('http://localhost:5620/app/synthetics/settings/params');
+    await page.click('text=No items found');
+    await page.click('button:has-text("Create Parameter")');
+    await page.click('[aria-label="Key"]');
+    await page.fill('[aria-label="Key"]', 'username');
+    await page.click('[aria-label="Value"]');
+    await page.fill('[aria-label="Value"]', 'elastic');
+    await page.click('.euiComboBox__inputWrap');
+    await page.fill('[aria-label="Tags"]', 'dev');
+    await page.click('[aria-label="Description"]');
+    await page.fill('[aria-label="Description"]', 'website username');
+    await page.click('button:has-text("Save")');
+    await page.click('text=website username');
+    await page.click('text=username');
+    await page.click('[aria-label="View parameter value"]');
+    await page.click('tbody >> text=elastic');
+    await page.click('[aria-label="View parameter value"]');
+    await page.click('text=•••••••');
```

Comment:
Should confirm that the username has the value expected.

Reply:
it already does few lines above